agent/dns/router.go

```diff
@@ -7,6 +7,7 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/consul/acl"
```

Comment:
micro-nit: import group is off. As an aside, I made the mistake of doing a `make tools` recently that installed a precommit hook for formatting these - so now I'm forced to change them back. Since this is a CE port, not offended if you leave it.